README.rst

```diff
@@ -5,3 +5,5 @@ drp_tasks
 ``drp_tasks`` is a package in the `LSST Science Pipelines <https://pipelines.lsst.io>`_.
 
 .. Add a brief (few sentence) description of what this package provides.
+
+# TODO: THIS NEEDS MORE TEXT!
```

Comment:
What's the story here?

Reply:
I put this in because the readme doesn't say anything about this package, but I don't actually know what the text should be since I'm not in DRP and didn't define it. I'll ask on slack, I guess, unless you have suggestions?